cmd/dep/main.go

```diff
@@ -56,7 +56,7 @@ type Config struct {
 // Run executes a configuration and returns an exit code.
 func (c *Config) Run() (exitCode int) {
 	// Build the list of available commands.
-	commands := []command{
+	commands := [...]command{
```

Comment:
What does this do?

Reply:
@ibrasho this is a statically typed array - just like `commands := [3]command{}` except the compiler will put that number for you so you don't have to update the number every time you change the array.

Reply:
👍